StorageUtility/Linux/StorageUtility.cpp

```diff
@@ -17,251 +17,181 @@ limitations under the License.
 */
 
 #include <libudev.h>
+#include <iostream>
+#include <fcntl.h>
+#include <map>
 #include <memory>
 #include <string.h>
+#include <unistd.h>
 
-#include "StorageUtility.h"
 #include "Device.h"
+#include "StorageUtility.h"
 
 namespace vtStor
 {
+std::map<String, vtStor::sStorageAdapterProperty> s_MappingSystemPathToProperties;
```

Comment:
At the application layer, I have 2 classes that use the vtStor. One is used to issue ATA command and one is used to issue ATA and SCSI command. How about if both of them do Enumerate drive?